vendor/ims.te

```diff
@@ -7,7 +7,8 @@ net_domain(ims)
 add_hwservice(ims, vnd_qti_ims_callinfo_hwservice)
 add_hwservice(ims, vnd_qti_ims_factory_hwservice)
 add_hwservice(ims, vnd_qti_imscms_hwservice)
-add_hwservice(ims, vnd_qti_uce_hwservice)
+
+hal_client_domain(ims, hal_telephony)
```

Comment:
```suggestion
hal_server_domain(ims, hal_telephony)
```

Reply:
You are correct, `imsrcsd` should also be a server domain for hal_telephony.

However, now `ims`, `rild` and `cnd` are all server domains. So much for separation of concerns, huh...

Reply:
@ix5 Corrrect. Just hacked together an idea for separating `ims` out again, but remains to be seen how many ties it has into the `radio` domain (qcrild)... https://github.com/MarijnS95/device-sony-sepolicy/compare/ims-attribute-group